inheritance/src/main/java/inheritance/Shop.java

```diff
@@ -0,0 +1,24 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Shop {
+    public String name;
+    public String description;
+    public String price;
+    public ArrayList<Review> reviews = new ArrayList<>();
+
+    public Shop(String name, String description, String price) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+    }
+
+    public void addReview(Review review) {
```

Comment:
Food for thought: could you have coded a way to prevent users from adding the same review more than once?